templates/overrides/semantic/web/Views/Home/Index.cshtml

```diff
@@ -0,0 +1,76 @@
+@{
+    ViewData["Title"] = "Home Page";
+}
+
+
+<div class="ui blue inverted vertical masthead center aligned segment">
+
+    <div class="ui text container">
+
+        <h1 class="ui inverted header">
+            MVC + Semantic UI
+        </h1>
+        <h2>If you build it, they will come.</h2>
+        <a class="ui huge secondary button" href="//semantic-ui.com/">
```

Comment:
This is wrong link. If content is hosted on HTTPS, it will fail, as semantic domain does not support HTTPS yet. Should be full link with protocol